packages/core-js/test/utils/hardhat/rpc.test.js

```diff
@@ -1,13 +1,25 @@
 const assert = require('assert/strict');
 const sinon = require('sinon');
-const { takeSnapshot, restoreSnapshot, fastForward } = require('../../../utils/hardhat/rpc');
+const {
+  takeSnapshot,
+  restoreSnapshot,
+  fastForward,
+  fastForwardTo,
+  getTime,
+} = require('../../../utils/hardhat/rpc');
 
 const fakeProvider = {
   send() {
     return new Promise((resolve) => {
       resolve(42);
     });
   },
+
+  getBlock() {
+    return new Promise((resolve) => {
+      resolve({ timestamp: 1337 });
+    });
+  },
```

Comment:
You can directly use the `async` interface for returning a Promise :)
```suggestion
  async getBlock() {
    return { timestamp: 1337 };
  },
```

Reply:
Implemented